code/API_definitions/most-frequent-location.yaml

```
@@ -0,0 +1,535 @@
+
+    This API can be used in anti-fraud cases where a service provider needs further evidence about the reported place of residence of the customer. Also, for service provision or personalization tasks when the service provider needs to manage their offer depending on the habits of the customer.
+
+    Such use cases are transversal to different sectors such as:
+
+      - Financial institutions: to mitigate risks about identity fraud, helping in their compliance with AML/KYC regulation.
+      - Insurance companies: to assess risks and prevent fraudulent claims.
+      - E-commerce and retail: to ensure accurate shipping and delivery, fraud prevention and targeted marketing.
+      - Digital Content Delivery companies: to verify service provisioning and billing, and to comply with regulation.
+      - Utilities: to verify service provisioning and billing, and to prevent service theft.
+      - Real estate: to screen potential tenants and ensure that lease agreements are in compliance with local laws.
+
+    # Relevant terms and definitions
+
+    * **Device**: Any physical entity that can connect to a network and participate in network communication.
+
+    * **Geographical reference**: Using coordinates or a reference code, the geographical surface where a device can be physically located is specified.
+
+    * **Verification**: The process triggered on the API server to calculate the frequency of time a device is at a given location.
+
+    # API functionality
+
+    The API allows customers to verify how often a device is in a specific location. The location can be specified using a geographical reference of type:
+      - `COVERAGE_ZONE`: It is specified by latitude and longitude coordinates. The coverage area is generated by one or more antennas closest to the specified coordinates, determining the device's connection to these antennas.
+      - `POSTAL_CODE`: A zip code or postal code that delimits a geographic area.
+
+    The operation returns a score representing how frequently the device was connected to antennas whose coverage cover the requested area in certain time slots. The range of score values is between "0" and "100", where higher scores indicate a more frequent presence in the location.
+
+    The `score` value maps to the frequency as follows:
+      - `score = 0`: The device has never been in the specified location.
+      - `score = 100`: The device is almost always in the specified location.
+
+    # Further info and support
+
+    (FAQs will be added in a later version of the documentation)
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0
+externalDocs:
+  description: Product documentation at CAMARA
+  url: https://github.com/camaraproject/
+servers:
+  - url: "{apiRoot}/most-frequent-location/v0"
```

According to https://wiki.camaraproject.org/display/CAM/API+versioning  the first introduction of a new API should have version "wip" 

```suggestion
  version: wip
externalDocs:
  description: Product documentation at CAMARA
  url: https://github.com/camaraproject/
servers:
  - url: "{apiRoot}/most-frequent-location/vwip"
```